packages/yargs/lib/webpack.js

```diff
@@ -17,32 +55,33 @@ exports.LoggerPlugin = class LoggerPlugin {
       if (this.lastHashes[name] === stats.hash) {
         return;
       }
-      this.lastHashes[name] = stats.hash;
+      const { hash, startTime, endTime } = stats;
+      const isRebuild = this.lastHashes[name];
+      const duration = prettyMS(endTime - startTime);
+
       const hasWarnings = stats.hasWarnings();
       const hasErrors = stats.hasErrors();
-      const duration = prettyMS(stats.endTime - stats.startTime);
+
+      const { assets, errors, warnings, children } = stats.toJson();
+
       if (hasErrors) {
-        this.logger.info(`build target '${name}' failed after ${duration}`);
+        this.logger.info(formatError(name, duration, isRebuild));
```

Comment:
`logger.`**`info`?** for an error?

Reply:
I consider this an informative message about the result of our compilation - the actual error (or warning) is being dealt with separately:

https://github.com/untool/untool/blob/029fa7242c201b41b8f00e7fdeceed2bdc79395e/packages/yargs/lib/webpack.js#L76-L83